### PR TITLE
OpenStack Configuration

### PR DESCRIPTION
**Pull request by akshaygin**
Included Network configuration and config file format

### CODE REVIEW

**Comment by akshaygin** [comment]
Completed issue #20

File: chapter2/openstack_config.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
-- login
-- Network config in OpenStack
-- config files
+Once the installation of OpenStack is done, we need to configure OpenStack Network to enable access to it's instances.
```
Comment (son-vyas) [inline chapter2/openstack_config.md]: " to access it's instances"  will work,  I guess.

**Review by KhushbuB — COMMENTED** [review]
Good work. You just need to add captions to the images.

File: chapter2/openstack_config.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+
+## Configure OpenStack Network
+
+Login into OpenStack dashboard using admin credentials. Go to **Project &rarr; Networks.** While on this page, hit on **Create Network** button and setup internal network as follows:
```
Comment (son-vyas) [inline chapter2/openstack_config.md]: It should be " Login to ...."
Hit **Create Network looks good , I guess

File: chapter2/openstack_config.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+![Create Router](../images/chapter2/Create_Router.png)
+
+Once router has been created, go to interface tab of that router and click on Add Interface. Select internal network and hit on Submit. We can see the whole network map in **Project &rarr; Network &rarr; Network Topology.**
+
+![Network Map](../images/chapter2/Network_Map.png)
+
+
+## OpenStack Configuration
+
+OpenStack has different components like Neutron, Nova etc., those are already mentioned in [chapter2.1](openstack_info.md). Each components uses similar kind of configuration techniques and _ini_ format for configuration files.
```
Comment (son-vyas) [inline chapter2/openstack_config.md]: I think "Each component uses..." looks good.

**Comment by akshaygin** [comment]
@son-vyas @KhushbuB Done with the changes.

**Comment by akshaygin** [comment]
Fix #20

**Pull request merged**